### PR TITLE
[bp/1.26] Backport stack (4)

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: ci/verify_examples.sh [hunk 1]
```diff
@@ -6,13 +6,25 @@ FAILED=()
 SRCDIR="${SRCDIR:-$(pwd)}"
 WARNINGS=()
 
+# Sandboxes listed here should be regarded as broken(/untested) until whatever
```
Comment (adisuissa) [inline ci/verify_examples.sh:9]: These were also broken in 1.26?

**Review by adisuissa — DISMISSED** [review]
LGTM, thanks!

File: ci/verify_examples.sh (hunk 1 above)
Comment (phlax) [reply ci/verify_examples.sh:9]: actually not `double-proxy` - just the other 2 ill remove now ...

Comment (phlax) [reply ci/verify_examples.sh:9]: updated

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**